core/store/migrations/migration1586956053/migrate.go

```diff
@@ -26,16 +26,6 @@ func Migrate(tx *gorm.DB) error {
 	UPDATE encrypted_secret_keys SET updated_at = '2019-01-01' WHERE updated_at IS NULL;
 	ALTER TABLE encrypted_secret_keys ALTER COLUMN updated_at SET NOT NULL;
 
-	ALTER TABLE encumbrances ADD COLUMN created_at timestamptz;
```

Comment:
what is an encumbrance?

Reply:
I have no idea, but it's only used in conjunction with service agreements.